Asterisk.2013/Asterisk.2013.sln

```diff
@@ -29,8 +31,8 @@ Global
 		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Release|Any CPU.Build.0 = Release|Any CPU
-		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Travis|Any CPU.ActiveCfg = Travis|Any CPU
-		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Travis|Any CPU.Build.0 = Travis|Any CPU
+		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Travis|Any CPU.ActiveCfg = Release|Any CPU
+		{BC6E7DBA-C05A-45FE-A2A3-B1637CE16274}.Travis|Any CPU.Build.0 = Release|Any CPU

```

Comment:
Why were these changed to `Release`? It might be why the Travis test is failing.

Reply:
Sorry, didn't pay attantion to addition configuration in solution. Fixed it.